src/vulkan-renderer/input/input.cpp

```diff
@@ -0,0 +1,72 @@
+#include "GLFW/glfw3.h"
+#include "spdlog/spdlog.h"
+#include <inexor/vulkan-renderer/input/input.hpp>
+
+namespace inexor::vulkan_renderer::input {
+void Input::key_callback(GLFWwindow * /*window*/, int key, int, int action, int /*mods*/) {
```

Comment:
```suggestion
void Input::key_callback(GLFWwindow * /*window*/, int key, int /*scancode*/, int action, int /*mods*/) {
```

Reply:
Ah yes: That's the standard way of saying "there once was a parameter" and also to mention the name of the parameter.